src/core/evaluator.js

```diff
@@ -1386,28 +1404,28 @@ class PartialEvaluator {
       const opArgs = operatorList.argsArray[lastIndex];
       opArgs[0].push(fn);
       Array.prototype.push.apply(opArgs[1], args);
-      minMax = opArgs[2];
-    }
-
-    // Compute min/max in the worker instead of the main thread.
-    // If the current matrix (when drawing) is a scaling one
-    // then min/max can be easily computed in using those values.
-    // Only rectangle, lineTo and moveTo are handled here since
-    // Bezier stuff requires to have the starting point.
-    switch (fn) {
-      case OPS.rectangle:
-        minMax[0] = Math.min(minMax[0], args[0], args[0] + args[2]);
-        minMax[1] = Math.max(minMax[1], args[0], args[0] + args[2]);
-        minMax[2] = Math.min(minMax[2], args[1], args[1] + args[3]);
-        minMax[3] = Math.max(minMax[3], args[1], args[1] + args[3]);
-        break;
-      case OPS.moveTo:
-      case OPS.lineTo:
-        minMax[0] = Math.min(minMax[0], args[0]);
-        minMax[1] = Math.max(minMax[1], args[0]);
-        minMax[2] = Math.min(minMax[2], args[1]);
-        minMax[3] = Math.max(minMax[3], args[1]);
-        break;
+      const minMax = opArgs[2];
+
+      // Compute min/max in the worker instead of the main thread.
+      // If the current matrix (when drawing) is a scaling one
+      // then min/max can be easily computed in using those values.
+      // Only rectangle, lineTo and moveTo are handled here since
+      // Bezier stuff requires to have the starting point.
+      switch (fn) {
+        case OPS.rectangle:
+          minMax[0] = Math.min(minMax[0], args[0], args[0] + args[2]);
+          minMax[1] = Math.max(minMax[1], args[0], args[0] + args[2]);
+          minMax[2] = Math.min(minMax[2], args[1], args[1] + args[3]);
+          minMax[3] = Math.max(minMax[3], args[1], args[1] + args[3]);
```

Comment:
Similar to the new code above, do you perhaps want to add similar `x`/`y` variables here as well to avoid recomputing those?
```suggestion
          const x = args[0] + args[2];
          const y = args[1] + args[3];
          minMax[0] = Math.min(minMax[0], args[0], x);
          minMax[1] = Math.max(minMax[1], args[0], x);
          minMax[2] = Math.min(minMax[2], args[1], y);
          minMax[3] = Math.max(minMax[3], args[1], y);
```